lib/ex_format.ex

```diff
@@ -294,7 +315,13 @@ defmodule ExFormat do
 
   # Variables
   def to_string({var, _, atom} = ast, fun) when is_atom(atom) do
-    fun.(ast, Atom.to_string(var))
+    zero_arity_fun? = Agent.get(:zero_arity_funs, &MapSet.member?(&1, var))
+    suffix =
+      case zero_arity_fun? do
+        true -> "()"
+        false -> ""
+      end
+    fun.(ast, Atom.to_string(var)) <> suffix
```

Comment:
Good catch! Seems like it's not a good idea to append the suffix to every single variable. What do you suggest to overcome this?